app/assets/javascripts/admin/spree/orders/variant_autocomplete.js.erb

```diff
@@ -52,39 +53,50 @@ $(document).ready(function() {
       }
       toggleItemEdit();
 
-      adjustItems(shipment_number, variant_id, quantity);
+      adjustItems(shipment_number, variant_id, quantity, true);
       return false;
     }
     $('a.save-item').click(handle_save_click);
 
-    handle_delete_click = function(elementSelector){
+    handle_delete_click = function(elementSelector, restock_item){
       var del = $(elementSelector);
       del.hide()
       var shipment_number = del.data('shipment-number');
       var variant_id = del.data('variant-id');
 
       toggleItemEdit();
 
-      adjustItems(shipment_number, variant_id, 0);
+       adjustItems(shipment_number, variant_id, 0, restock_item);
```

Comment:
What's this space doing here? :thinking:

Reply:
do we have lint for js files that we can run?

Reply:
Apparently we don't have that yet. And this is an ERB file evaluated to JS. Linting can only happen when it's compiled. So maybe there could be some tool that's integrating with webpacker to lint the file first? Then we could do that on CI.